src/ThreeEditor/main.js

```diff
@@ -48,7 +49,14 @@ export function initEditor(container) {
 			if (isLoadingFromHash) return;
 
 			if (typeof state !== 'undefined') {
-				editor.fromJSON(state);
+				let versionIsOk = true;
+				if (state.metadata.version !== editor.jsonVersion) {
+					versionIsOk = window.confirm(
+						`Editor in memory has version of JSON: ${state.metadata.version}\nCurrent version of editor JSON: ${editor.jsonVersion}\nLoad it anyway?`
```

Comment:
Shouldn't this be consistent with:

![image](https://user-images.githubusercontent.com/7374733/144467162-49e2b11f-a7bc-4891-824d-88f32b9feb6b.png)

?

Reply:
One message is for loading form file one for loading from browser database.

Reply:
OK